server/models/profileModel.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+
+const profileSchema = mongoose.Schema (
+    {   
```

Comment:
can we add name to this schema/

Reply:
@meyeralex31  Thanks fixed it now https://github.com/meyeralex31/csci550/pull/1/commits/d3776e95720c63e060e81d2879e4e94ec4b2d176